.travis.yml

```diff
@@ -17,12 +17,12 @@ matrix:
           - extra-android-support
           - sys-img-armeabi-v7a-android-24
       script:
-        - ./gradlew clean build -x signArchives -x test -x checkstyleTest
```

Comment:
checkstyle only brings pain while writing tests. I've actually disabled it for tests, because having tests is more important than having tests perfectly indented/documented/whatever.

Reply:
makes sense, I've reverted the changes